### PR TITLE
[8.0] Add tree view and manage datetime of last materialized view refreshment

### PR DESCRIPTION
**Pull request by legalsylvain**
* ADD tree view management. courtesy @jbeficent. https://twitter.com/jbeficent_erp/status/885587899424681986

* ADD datetime of the last refresh of the materialized view

* improve fr translation

* improve display of list of view, displaying color depending of the state of the view. (blue = draft, etc..)

Supersed #149

### CODE REVIEW

File: bi_sql_editor/models/bi_sql_view.py [hunk 1]
```diff
@@ -218,10 +226,9 @@ def button_open_view(self):
         return {
             'type': 'ir.actions.act_window',
             'res_model': self.model_id.model,
-            'view_id': self.graph_view_id.id,
             'search_view_id': self.search_view_id.id,
-            'view_type': 'graph',
-            'view_mode': 'graph',
+            'view_type': 'form',
```
Comment (bealdav) [inline bi_sql_editor/models/bi_sql_view.py:231]: No blocking from me but 'view_type' is no more required/used: it works without it

**Review by bealdav — APPROVED** [review]

File: bi_sql_editor/models/bi_sql_view.py (hunk 1 above)
Comment (legalsylvain) [reply bi_sql_editor/models/bi_sql_view.py:231]: hi @bealdav, view_type is used until last branches. (V10).
https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_actions.py#L295
but yes, view_type has a default value to 'form'. 

I propose to keep this code, and better to make a batch to remove globally, all that useless keys, in all OCA repository.

regards.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

File: bi_sql_editor/models/bi_sql_view.py (hunk 1 above)
Comment (bealdav) [reply bi_sql_editor/models/bi_sql_view.py:231]: Thanks for the precision

**Comment by legalsylvain** [comment]
merging. (2 :+1: and working on production).

**Pull request merged**